cmd/zc_traverser_local.go

```diff
@@ -46,6 +51,10 @@ type localTraverser struct {
 	// a generic function to notify that a new stored object has been enumerated
 	incrementEnumerationCounter enumerationCounterFunc
 	errorChannel                chan ErrorFileInfo
+
+	targetHashType common.SyncHashType
+	// receives fullPath entries and manages hashing of files lacking metadata.
+	hashTargetChannel chan string
```

Comment:
so this channel is used for sending local paths that need hashing?

Reply:
Correct.